index.js

```diff
@@ -70,16 +67,17 @@ if (!isProduction) {
 
 // production error handler
 // no stacktraces leaked to user
-app.use(function (err, req, res, next) {
+app.use((err, req, res) => {
```

Comment:
you omitted the `next` parameter in the refactored middleware.

Reply:
Apart from Resolving the hound errors and including the next parameter as stated in the comment above. I believe the PR is good to go.